Assets/_Scripts/GameStatus.cs

```diff
@@ -3,36 +3,142 @@
 
 public class GameStatus : MonoBehaviour {
 
+	static public GameStatus Instance;
+
+
+	// LoseLife: You will die when you lose enough lives;
+	// GainScore: teamwork. Your death will increase the other team's Score
+	public enum GameMode{LoseLife, GainScore};
+	public GameMode gameMode;
+	// Target lives/scores in each mode for winning
+	public int GameTargetRounds = 2;
+	public GameObject playerPrefab;
+
 	// private int playerNum;
 	private bool isGameOver = false;
+	private int[] playerDeathCount = new int[4]{0,0,0,0};
+
+	private UserData[] userDataCollection = new UserData[4];
+
+	void Awake()
+	{
+		Instance = this;
+	}
 	// Use this for initialization
 	void Start () {
 		// playerNum = GameObject.FindGameObjectsWithTag (TagList.Player).Length;
+		BindPlayerData ();
+	}
+
+	void BindPlayerData()
```

Comment:
Do you mean bind players data?
If you do a for loop inside. Use Players can be more intuitive